.vscode/settings.json

```diff
@@ -73,6 +73,7 @@
         "set": "cpp",
         "stack": "cpp",
         "unordered_set": "cpp",
+        "filesystem": "cpp"
```

Comment:
I think there is a merge conflict here that you will have to fix before merging, I don't think it matters which version you pick to keep.

Reply:
It was a missing comma after the "cpp" that was missing. Fixed